zebROS_ws/src/controllers_2023/src/elevator_controller.cpp

```diff
@@ -118,9 +119,78 @@ bool ElevatorController_2023::init(hardware_interface::RobotHW *hw,
     //create the interface used to initialize the talon joint
     hardware_interface::TalonCommandInterface *const talon_command_iface = hw->get<hardware_interface::TalonCommandInterface>();
 
-    //hardware_interface::PositionJointInterface *const pos_joint_iface = hw->get<hardware_interface::PositionJointInterface>();
+    //hardware_interface::PositionJointInterface *const pos_joint_iface = hw->get<hardware_interface::PositionJointInterface>()
+    if (!readIntoScalar(controller_nh, "arb_feed_forward_low", arb_feed_forward_up_low))
```

Comment:
Since we don't have an up (normal elevator) vs down (climbing) distinction anymore, remove the _up from these names to avoid confusion later on